pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[build-system]
+requires = ["setuptools>=59.0"]
```

Comment:
Why do we need setuptools>=59.0? And should we add "wheel"?

Reply:
We don't need this file at all, I had added it to try to avoid the distutils deprecation warning, removing it